cmd/kubeless/ingress.go

```diff
@@ -31,7 +31,7 @@ func init() {
 
 	for _, cmd := range cmds {
 		ingressCmd.AddCommand(cmd)
-		cmd.Flags().StringP("namespace", "n", "", "Specify namespace for the ingress")
+		cmd.Flags().StringP("namespace", "n", "", "Specify namespace for the route")
```

Comment:
Not related to this PR. But things like this (discussed in #479) are confusing. Route is abstraction for user to specify how http endpoint should be. Use of namespace does not make sense here.

Reply:
well, it makes sense to identify the function, it is not the same `route add myfunc -n ns1` than  `route add myfunc -n ns2` since those two functions may be different (thinking on multi-tenancy). I would just change the description to `Specify namespace of the target function`.

Reply:
> it is not the same route add myfunc -n ns1 than route add myfunc -n ns2 since those two functions may be different

Should function user really care if Kubeless can not figure out function is from which namespace, how to handle conflict if same function name exist in two different namespace. With this command all user wants is how function endpoint URL should look like.

My argument has been use of Kubernetes for Kubeless is implementation detail, unless we want  Kubeless to have Kubectl'ish (or native Kubernetes) experiance.

Even with kubectl use of `--namespace` is for convenience to override the current set namespace in the kubecfg context.

Reply:
> My argument has been use of Kubernetes for Kubeless is implementation detail, unless we want Kubeless to have Kubectl'ish (or native Kubernetes) experiance.
> Even with kubectl use of --namespace is for convenience to override the current set namespace in the kubecfg context.

Yes, as I see this, mimicking `kubectl` behavior is something desirable. It is true that this should work by default (without specifying the `--namespace` flag) if everything was used always by default. In my opinion that doesn't mean that we should prevent the users from advanced features. (If not we will be putting stones in our way to support for example multi-tenancy).